ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -907,5 +954,67 @@ public static void calculateGoTermFDR(List<GoTermNumber> gtns)
 
         #endregion GO-TERMS AND GO-TERM SIGNIFICANCE
 
+        #region CLEAR METHODS
+
+        public void clear_et()
+        {
+            SaveState.lollipop.et_relations.Clear();
+            SaveState.lollipop.et_peaks.Clear();
+            SaveState.lollipop.ed_relations.Clear();
+            SaveState.lollipop.target_proteoform_community.families.Clear();
+            SaveState.lollipop.decoy_proteoform_communities.Values.Select(c => c.families).ToList().Clear();
+
+            foreach (ProteoformCommunity community in decoy_proteoform_communities.Values.Concat(new List<ProteoformCommunity> { target_proteoform_community }))
+            {
+                foreach (Proteoform p in community.experimental_proteoforms)
+                {
+                    p.relationships.RemoveAll(r => r.RelationType == ProteoformComparison.ExperimentalTheoretical || r.RelationType == ProteoformComparison.ExperimentalDecoy);
+                    p.family = null;
+                    p.ptm_set = new PtmSet(new List<Ptm>());
+                    p.linked_proteoform_references = null;
+                    p.gene_name = null;
+                }
+
+                foreach (Proteoform p in community.theoretical_proteoforms)
+                {
+                    p.relationships.RemoveAll(r => r.RelationType == ProteoformComparison.ExperimentalTheoretical || r.RelationType == ProteoformComparison.ExperimentalDecoy);
+                    p.family = null;
+                }
+
+                et_peaks.RemoveAll(k => k.RelationType == ProteoformComparison.ExperimentalTheoretical || k.RelationType == ProteoformComparison.ExperimentalDecoy);
```

Comment:
Could you extend the test for this method and the EE one to include checking whether ET peaks are removed appropriately?